tempesta_fw/t/unit/sched_helper.c

```diff
@@ -95,19 +95,21 @@ test_create_srv(const char *in_addr, TfwSrvGroup *sg)
 
 	return srv;
 }
-
 TfwSrvConnection *
 test_create_conn(TfwPeer *peer)
 {
+	void (*tfw_connection_link_peer)(TfwConnection *conn,TfwPeer *peer);
 	static struct sock __test_sock = {
 		.sk_state = TCP_ESTABLISHED,
 	};
 	TfwSrvConnection *srv_conn;
 
+	tfw_connection_link_peer = get_sym_ptr("tfw_connection_link_peer");
+
 	BUG_ON(tfw_srv_conn_alloc_ptr == NULL);
 	srv_conn = tfw_srv_conn_alloc_ptr();
 	BUG_ON(!srv_conn);
-
+	BUG_ON(tfw_connection_link_peer == NULL);
```

Comment:
Wrong deletion of empty line and it's better to place the new BUG_ON above the empty line near from previous BUG_ON